tests/components/devolo_home_control/test_init.py

```diff
@@ -63,6 +64,21 @@ async def test_unload_entry(hass: HomeAssistant) -> None:
         assert entry.state is ConfigEntryState.NOT_LOADED
 
 
+async def test_home_assistant_stop(hass: HomeAssistant) -> None:
+    """Test home assistant stop."""
+    entry = configure_integration(hass)
+    test_gateway = HomeControlMock()
+    with patch(
+        "homeassistant.components.devolo_home_control.HomeControl",
+        side_effect=[test_gateway, HomeControlMock()],
```

Comment:
```suggestion
    test_gateway = HomeControlMock()
    test_gateway2 = HomeControlMock()
    with patch(
        "homeassistant.components.devolo_home_control.HomeControl",
        side_effect=[test_gateway, test_gateway2],
```
Let's test that `websocket_disconnect` is called for both gateways.

Reply:
The second gateway is more or less just a prove, that the hole process also works if a customer own more than one gateway. But testing it explicitly also doesn't hurt :+1: